internal/pkg/api/error.go

```diff
@@ -74,10 +74,10 @@ func NewHTTPErrResp(err error) HTTPErrResp {
 		{
 			context.Canceled,
 			HTTPErrResp{
-				http.StatusServiceUnavailable,
-				"ServiceUnavailable",
+				http.StatusClientClosedRequest,
+				"StatusClientClosedRequest",
 				"server is stopping",
-				zerolog.DebugLevel,
+				zerolog.WarnLevel,
```

Comment:
If we are sure this only will happen in cases of client disconnects, maybe info is fine

Reply:
updated